CMakeLists.txt

```diff
@@ -171,11 +171,11 @@ file(WRITE ${CMAKE_CURRENT_BINARY_DIR}/libvast/vast/documentation.hpp
 
 # Choose a deafult log level based on build type.
 if (CMAKE_BUILD_TYPE STREQUAL Release)
-  set(VAST_LOG_LEVEL_DEFAULT "VERBOSE")
+  set(VAST_LOG_LEVEL_DEFAULT "DEBUG")
 elseif (CMAKE_BUILD_TYPE STREQUAL MinSizeRel)
-  set(VAST_LOG_LEVEL_DEFAULT "VERBOSE")
-elseif (CMAKE_BUILD_TYPE STREQUAL RelWithDebInfo)
   set(VAST_LOG_LEVEL_DEFAULT "DEBUG")
+elseif (CMAKE_BUILD_TYPE STREQUAL RelWithDebInfo)
+  set(VAST_LOG_LEVEL_DEFAULT "TRACE")
```

Comment:
Not sure about this part, `RelWithDebInfo` is still a release build (probably even the one that will be distributed) so it doesnt seem right to include a performance penalty if you select it.

I'd just have the same log level for all the release builds, for consistency.

Reply:
We distribute `Release` builds, not `RelWithDebInfo`.

I have no strong opinion on this, so I'll change it back.